src/components/Graph.react.js

```diff
@@ -516,11 +533,43 @@ const graphPropTypes = {
          */
         modeBarButtons: PropTypes.any,
 
+        /**
+         *
+         */
+        toImageButtonOptions: PropTypes.shape({
+            /**
+             * The file format to create
+             */
+            format: PropTypes.oneOf(['jpeg', 'png', 'webp', 'svg']),
+            /**
+             * The name given to the downloaded file
+             */
+            filename: PropTypes.string,
+            /**
+             * Width of the downloaded file, in px
+             */
+            width: PropTypes.number,
+            /**
+             * Height of the downloaded file, in px
+             */
+            height: PropTypes.number,
+            /**
+             * Extra resolution to give the file after
+             * rendering it with the given width and height
+             */
+            scale: PropTypes.number,
+        }),
+
         /**
          * add the plotly logo on the end of the mode bar
          */
         displaylogo: PropTypes.bool,
 
+        /**
+         * add the plotly logo even with no modebar
```

Comment:
```suggestion
         * Add the plotly logo even with no modebar
```

Reply:
Ah right, a lot of the older docstrings didn't bother capitalizing so I tried to follow that, but then some of the newer ones do... I should just go through and capitalize them all. Thanks for pointing this out.